infra/label_evaluator_step_functions/lambdas/unified_receipt_evaluator.py

```
@@ -0,0 +1,1198 @@
+            # LEGACY: Load from S3 (for backwards compatibility)
+            logger.info(
+                "Loading receipt data from s3://%s/%s",
+                batch_bucket,
+                data_s3_key,
+            )
+            target_data = load_json_from_s3(
+                s3, batch_bucket, data_s3_key, logger=logger
+            )
+
+            if target_data is None:
+                raise ValueError(f"Receipt data not found at {data_s3_key}")
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)